homeassistant/components/lock/mqtt.py

```diff
@@ -180,8 +182,10 @@ def assumed_state(self):
         This method is a coroutine.
         """
         mqtt.async_publish(
-            self.hass, self._command_topic, self._payload_lock, self._qos,
-            self._retain)
+            self.hass, self._config.get(CONF_COMMAND_TOPIC),
```

Comment:
Use `dict[key]` for required config keys and keys with default config schema values.